### PR TITLE
fix "undo" style in delete note alert; change example className to ma…

### PR DESCRIPTION
**Pull request by Brick-Mover**
…tch apm style

### CODE REVIEW

**Comment by Brick-Mover** [comment]
[Change flash message color and undo UX for notes block](https://trello.com/c/Q3kNCXax/1499-change-flash-message-color-and-undo-ux-for-notes-block)

**Review by MMullin7 — APPROVED** [review]

File: src/components/DeletedNote.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const DeletedNote: React.FunctionComponent<DeletedNoteProps> = ({
       <span className="align-middle">Note deleted.</span>
       {onUndelete &&
         <Button color="link" onClick={() => onUndelete(note)} className="ml-1 p-0">
-          undo
+          <u>Undo</u>
```
Comment (gthomas-appfolio) [inline src/components/DeletedNote.tsx:23]: Why is this a u tag?  Seems odd

**Review by gthomas-appfolio — COMMENTED** [review]
One question

File: src/components/DeletedNote.tsx (hunk 1 above)
Comment (Brick-Mover) [reply src/components/DeletedNote.tsx:23]: Yeah so we want to add underscore to "Undo" button to make it more obvious

**Pull request merged**